pom.xml

```diff
@@ -6216,7 +6217,15 @@
         <artifactId>javaparser-core</artifactId>
         <version>${version.com.github.javaparser}</version>
       </dependency>
+      
+      <!-- Used by Test Scenario for Thread sleep -->
+	<dependency>
+    	<groupId>org.awaitility</groupId>
+    	<artifactId>awaitility</artifactId>
```

Comment:
Do we even need to use this library? Why not use just a simple `CountDownLatch`.

Reply:
Please answer @MarianMacik's question too